### PR TITLE
commit on 'code-journey'

### PR DESCRIPTION
**Pull request by Demilade-Ayoola**
I added my code journey, followed the guidelines and also added my data to the Data.json

I made sure of the following:
-  My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
-  My changes generate no new warnings
-  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by KendallDoesCoding — COMMENTED** [review]
You haven't added your data in the Data.json file!

**Comment by Demilade-Ayoola** [comment]
> 

I have commited the changes in the Data.Json file

**Comment by KendallDoesCoding** [comment]
> > 
> 
> I have commited the changes in the Data.Json file

Thank you!

File: journeys/Demilade-Ayoola.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# <h1 align="center">[Demilade-Ayoola](https://github.com/Demilade-Ayoola)' journey</h1>
+<h2 align="center">Front-end developer from Nigeria</h2>
+
+- My name is Demilade and i'm from Lagos, Nigeria, I am a front-end web developer, A demographer and Data analyst
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:4]:
```suggestion
- My name is Demilade and I'm from Lagos, Nigeria. I am a front-end web developer. A demographer and Data analyst.
```

File: journeys/Demilade-Ayoola.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- My hobbies include travelling, coding, and playing sports like Tennis and Soccer
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:6]:
```suggestion
- My hobbies include traveling, coding, and playing sports like Tennis and Soccer.
```

File: journeys/Demilade-Ayoola.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- I am currently learning a couple of programming languages and using them to build projects to profer solutions and solve problems
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:8]:
```suggestion
- I am currently learning a couple of programming languages and using them to build projects to proper solutions and solve problems.
```
```

File: journeys/Demilade-Ayoola.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## My Journey
+- I started learning coding in june 2021, when i wrote my first "Hello World"
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:11]:
```suggestion
- I started learning coding in June 2021, when I wrote my first "Hello World".
```

File: journeys/Demilade-Ayoola.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- I was taught HTML, CSS, Bootstrap within two months which i designed some projects. My learning course was interrupted for about 5 months due to my academics, but i returned back to continue learning
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:13]:
```suggestion
- I was taught HTML, CSS, and Bootstrap within two months, of which I designed some projects. My learning course was interrupted for about 5 months due to my academics, but I returned back to continue learning.
```

File: journeys/Demilade-Ayoola.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- My hobbies include travelling, coding, and playing sports like Tennis and Soccer
+
+- I am currently learning a couple of programming languages and using them to build projects to profer solutions and solve problems
+
+## My Journey
+- I started learning coding in june 2021, when i wrote my first "Hello World"
+
+- I was taught HTML, CSS, Bootstrap within two months which i designed some projects. My learning course was interrupted for about 5 months due to my academics, but i returned back to continue learning  
+
+- I also learnt JavaScript and worked on some projects like Beat-maker project,Weather app, a travel website, To-do List. I learnt how to make animations using GSAP library and Scrollmagic which i used effectively in the travel site
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:15]:
```suggestion
- I also learned JavaScript and worked on some projects like Beat-maker project, Weather app, a travel website, To-do List. I learned how to make animations using GSAP library and Scrollmagic which i used effectively in the travel site.
```

File: journeys/Demilade-Ayoola.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+- I am currently learning a couple of programming languages and using them to build projects to profer solutions and solve problems
+
+## My Journey
+- I started learning coding in june 2021, when i wrote my first "Hello World"
+
+- I was taught HTML, CSS, Bootstrap within two months which i designed some projects. My learning course was interrupted for about 5 months due to my academics, but i returned back to continue learning  
+
+- I also learnt JavaScript and worked on some projects like Beat-maker project,Weather app, a travel website, To-do List. I learnt how to make animations using GSAP library and Scrollmagic which i used effectively in the travel site 
+
+- I am currently learning React-js and I have built a Banking App and also an E-commerce app.
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:17]:
```suggestion
- I am currently learning React-js and I have built a Banking App and also an E-commerce App.
```

File: journeys/Demilade-Ayoola.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+- I am currently looking for an internship oppotunities to get more knowledge and experience as a Software Developer
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:19]:
```suggestion
- I am currently looking for an internship opportunities to get more knowledge and experience as a Software Developer.
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
fix typos, grammer.

**Comment by KendallDoesCoding** [comment]
Please commit changes requested at the earliest @Demilade-Ayoola

**Comment by Demilade-Ayoola** [comment]
I have commited the changes in the typo and errors

File: journeys/Demilade-Ayoola.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+- My hobbies include traveling, coding, and playing sports like Tennis and Soccer.
+
+- I am currently learning a couple of programming languages and using them to build projects to proffer solutions and solve problems.
+
+## My Journey
+- I started learning coding in June 2021, when I wrote my first "Hello World".
+
+- I was taught HTML, CSS and Bootstrap within two months, of which I designed some projects. My learning course was interrupted for about 5 months due to my academics, but I returned back to continue learning.  
+
+- I also learned JavaScript and worked on some projects like Beat-maker project,Weather app, a travel website, To-do List. I learned how to make animations using GSAP library and Scrollmagic which i used effectively in the travel site.
```
Comment (KendallDoesCoding) [inline journeys/Demilade-Ayoola.md:15]:
```suggestion
- I also learned JavaScript and worked on some projects like Beat-maker project,Weather app, a travel website, To-do List. I learned how to make animations using GSAP library and Scrollmagic which I used effectively in the travel site. 
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by KendallDoesCoding** [comment]
@Demilade-Ayoola Please resolve conflicts in Data.json

**Review by KendallDoesCoding — APPROVED** [review]

**Pull request merged**